### DISCUSSION TITLE
help | Boost a href without scroll to top

### DISCUSSION THREAD

**Discussion by zipper-x** [bigskysoftware/htmx#1165] [Q&A] [answered]
Hello there)
i boost my links with hx-boost="true" 
but i want to disable scroll to top after click,
any solutions or recommendations?)

```
<div hx-boost="true">
	 <a href="/?genre=all">ALL</a>
         <a href="/?genre=other">other</a>
```

**Comment by David-Guillot** [comment, marked as answer]
Hi @zipper-x ,

The `hx-swap` [docs](https://htmx.org/attributes/hx-swap/) shows plenty of options to control the scroll behavior. Have you tried playing with it?

**Reply by zipper-x** [reply]
Yes, that helped a bit, the thing is that from the beginning i was doing wrong, need to write another code)
Thank you)

**Comment by gone** [comment]
You might want to configure the "scrollBehavior" config setting - default is smooth scroll to the top, the other option is "auto" which behaves like a normal link
